src/test/scala/org/scalastyle/scalariform/NonASCIICharacterCheckerTest.scala

```diff
@@ -39,8 +39,18 @@ class NonASCIICharacterCheckerTest extends AssertionsForJUnit with CheckerTest {
     assertErrors(List(), source)
   }
 
+  @Test def testStringOK(): Unit = {
+    val source = """
+                   |package foobar
+                   |// non-ascii in string via unicode escape - ok
+                   |class OK {
+                   |  val s = "%s"
+                   |}""".stripMargin.format("\\u1f34e")
```

Comment:
If somebody knows how to correctly produce the desired string in a cleaner way I'd be happy to update.

I found that using `\u1f34e` in triple-quote string resulted in unicode literal (i.e. escape was applied), and using `\\u1f34e` resulted in literal double slash. Unsure how to cleanly get desired outcome of literal `\u1f34e` within a triple-quote string.